src/routes/+page.svelte

```diff
@@ -203,6 +270,104 @@
   {$_("contact_us")}
 </h1>
 
+<div class="mb-20 flex flex-col items-center gap-2">
+  <form
+    class="w-full max-w-xl"
+    method="POST"
+    on:submit|preventDefault={handleSubmit}
+  >
+    <div class="mb-6 md:flex md:items-center">
+      <div class="md:w-1/3">
+        <label
+          class="mb-1 block pr-4 font-bold text-gray-500 md:mb-0 md:text-right"
+          for="name"
+        >
+          Your name
+        </label>
+      </div>
+      <div class="md:w-2/3">
+        <input
+          class="w-full appearance-none rounded border-2 border-gray-200 bg-gray-200 px-4 py-2 leading-tight text-gray-700 focus:border-purple-500 focus:bg-white focus:outline-none"
+          id="name"
+          name="name"
+          required
+          type="text"
+          bind:value={name}
+        />
+      </div>
+    </div>
+
+    <div class="mb-6 md:flex md:items-center">
+      <div class="md:w-1/3">
+        <label
+          class="mb-1 block pr-4 font-bold text-gray-500 md:mb-0 md:text-right"
+          for="email"
+        >
+          Your email
+        </label>
+      </div>
+      <div class="md:w-2/3">
+        <input
+          class="w-full appearance-none rounded border-2 border-gray-200 bg-gray-200 px-4 py-2 leading-tight text-gray-700 focus:border-purple-500 focus:bg-white focus:outline-none"
+          id="email"
+          name="email"
+          required
+          type="email"
+          bind:value={email}
+        />
+      </div>
+    </div>
+
+    <div class="mb-6 md:flex md:items-center">
+      <div class="md:w-1/3">
+        <label
+          class="mb-1 block pr-4 font-bold text-gray-500 md:mb-0 md:text-right"
+          for="msg"
+        >
+          Message
```

Comment:
```suggestion
          {$_("form.message")}
```

Y declarar la traducción en los dos idiomas que tenemos (`en.json` y `es.json`)